src/restaurants/restaurants_CRUD.go

```diff
@@ -0,0 +1,104 @@
+package restaurants
+
+import (
+	"github.com/Nastya-Kruglikova/cool_tasks/src/services/common"
+	"github.com/gorilla/mux"
+	"github.com/satori/go.uuid"
+	"net/http"
+	"strconv"
+)
+
+type successCreate struct {
+	Status string     `json:"message"`
+	Result Restaurant `json:"result"`
+}
+
+type successDelete struct {
+	Status string `json:"message"`
+}
+
+func Get(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.Query()
+	if val, ok := query["id"]; ok {
+		id, err := uuid.FromString(val[0])
+		if err != nil {
+			common.SendNotFound(w, r, "ERROR: Invalid ID", err)
+			return
+		}
+		items, err := getByID(id)
+
+		if err != nil {
+			common.SendNotFound(w, r, "ERROR: Can't get items", err)
+			return
+		}
+
+		common.RenderJSON(w, r, items)
+	}
+
+
+	//MAGIC BEGINS!!!
+	items, err := getByQuery(query)
+	//MAGIC ENDS!!!
+	if err != nil {
+		common.SendNotFound(w, r, "ERROR: Can't get items", err)
+		return
+	}
+
+	common.RenderJSON(w, r, items)
+}
+
+func PostRestaurant(w http.ResponseWriter, r *http.Request) {
```

Comment:
Get, Delete, but PostRestaurant ? Pls be consistent with function names. 
Moreover, restaurants.PostRestaurant() is too long and not DRY - (don`t repeat yourself) principle